scrapy_autounit/utils.py

```diff
@@ -280,14 +278,16 @@ def generate_test(fixture_path, encoding='utf-8'):
     spider_cls.update_settings(settings)
     for k, v in data.get('settings', {}).items():
         settings.set(k, v, 50)
+
     crawler = Crawler(spider_cls, settings)
-    spider = spider_cls.from_crawler(crawler, **data.get('spider_args'))
+    spider = spider_cls.from_crawler(crawler, **data.get('spider_args_in'))
```

Comment:
I just ran into an issue trying these changes out.
If you have legacy fixtures (prior to these changes) you would need to generate them all over again (because they miss `spider_args_in`).
Maybe we can look for `spider_args` first and if not present try with the new `spider_args_in`.
Also adding a `{}` as default to `get` would be nice, because no default leads to `**None` which will make python break.